integrations/milvus-document-store.md

```diff
@@ -33,60 +34,92 @@ toc: true
 ---
 
 ### Installation
-```console
-pip install -U milvus-haystack
+
+```shell
+pip install --upgrade pymilvus milvus-haystack
 ```
 
 ### Usage
 
-First, to start up a Milvus service, follow the ['Start Milvus'](https://milvus.io/docs/install_standalone-docker.md#Start-Milvus) instructions in the documentation. 
+Use the `MilvusDocumentStore` in a Haystack pipeline as a quick start.
+
+```python
+from haystack import Document
+from milvus_haystack import MilvusDocumentStore
+
```

Comment:
Running into this code, there's a dependency error:

```
ContextualVersionConflict                 Traceback (most recent call last)
[<ipython-input-2-678eb02a0f80>](https://localhost:8080/#) in <cell line: 2>()
      1 from haystack import Document
----> 2 from milvus_haystack import MilvusDocumentStore
      3 
      4 document_store = MilvusDocumentStore(
      5     connection_args={"uri": "./milvus.db"},  # Milvus Lite

8 frames
[/usr/local/lib/python3.10/dist-packages/pkg_resources/__init__.py](https://localhost:8080/#) in _resolve_dist(self, req, best, replace_conflicting, env, installer, required_by, to_activate)
    871             # Oops, the "best" so far conflicts with a dependency
    872             dependent_req = required_by[req]
--> 873             raise VersionConflict(dist, req).with_context(dependent_req)
    874         return dist
    875 

ContextualVersionConflict: (grpcio 1.64.0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('grpcio<=1.63.0,>=1.49.1'), {'pymilvus'})
```

@zc277584121 any ideas on how to resolve? This was in a Colab, in a clean Python environment where I ran only the pip commands suggested in this documentation.

Reply:
In Colab, after running the pip command, we have to restart the session kernel to make the grpcio dependency really installed. I think it is a problem in Colab, where the default environment is not clean but exist an installed grpcio. And after pip install pymilvus[->grpcio], actually it is not be install and can not work normally. and we have to restart the colab kernel to make the `pip install xxx` really be installed and work normally.

Reply:
here is our official document: https://milvus.io/docs/integrate_with_haystack.md , where we mind users to restart kernel in colab
![image](https://github.com/deepset-ai/haystack-integrations/assets/17022025/2ef5ee67-527c-4598-ade4-545527fb6639)

Reply:
I added a suggestion in this page, too